src/main/java/com/onarandombox/MultiversePortals/PortalPlayerSession.java

```diff
@@ -257,24 +257,53 @@ public boolean showDebugInfo(MVPortal portal) {
     }
 
     private void showPortalPriceInfo(MVPortal portal) {
-        getPlayerFromName().sendMessage("More details for you: " + ChatColor.GREEN + portal.getDestination());
         if (portal.getPrice() > 0D) {
             getPlayerFromName().sendMessage("Price: " + ChatColor.GREEN + plugin.getCore().getEconomist().formatPrice(portal.getPrice(), portal.getCurrency()));
         } else if (portal.getPrice() < 0D) {
-            getPlayerFromName().sendMessage("Prize: " + ChatColor.GREEN + plugin.getCore().getEconomist().formatPrice(-portal.getPrice(), portal.getCurrency()));
+            getPlayerFromName().sendMessage("Price: " + ChatColor.GREEN + plugin.getCore().getEconomist().formatPrice(-portal.getPrice(), portal.getCurrency()));
         } else {
             getPlayerFromName().sendMessage("Price: " + ChatColor.GREEN + "FREE!");
         }
     }
 
     public static void showStaticInfo(CommandSender sender, MVPortal portal, String message) {
```

Comment:
Honestly, I hate the fact that this method is even here. Can we just revert this to the original form and `@deprcated` it? Then in the `InfoCommand` class, we move all logic of parsing nicely formated text there instead of calling this method.

Reply:
Thanks, this was my first thought, but I was too lazy before ;-)

Reply:
Oh, wouldn't it be better to have a own `Display` class to handle all the messages sent to a player?
There would be a static variable plugin to handle the plugin's instance and the static methods to display the parametered portal.
I think the showStaticInfo method is not placed correctly in the `InfoCommand` class.

Reply:
Yea in the long term that what I hope _will_ happen, you can see https://github.com/Multiverse/Multiverse-Core/pull/2584

But it's ok for now for the scope of this PR